data/json/items/comestibles.json

```diff
@@ -2620,8 +2620,8 @@
         "price" : 500,
         "material" : "flesh",
         "volume" : 1,
-        "charges" : 1,
-        "stack_size" : 2,
+        "stack_size" : 4,
```

Comment:
I don't PR before testing.

Four charges fit into one point of volume, eight into two.